src/routes/safe_apps/models.rs

```diff
@@ -11,6 +11,7 @@ pub struct SafeApp {
     pub description: String,
     pub chain_ids: Vec<String>,
     pub provider: Option<SafeAppProvider>,
+    pub access_control: SafeAppAccessControlPolicies,
```

Comment:
This may have to be `Option<SafeAppAccessControlPolicies>` too if the value from the backend is also expected to be nullable. In which case I would suggest adding `#[serde(skip_serializing_if = "Option::is_none")]
` to the field.